### PR TITLE
Improved SimpleScoreboard efficiency

### PR DESCRIPTION
**Pull request by Tvhee-Dev**
- Improved the copyColors() method to split the text quicker with less performance!
- Removed Object for lines, you are forced to set your line as a String now!

### CODE REVIEW

**Comment by Tvhee-Dev** [comment]
Ok, so I have removed all the tab characters and replaced them with spaces. Then I have also updated the copyColors() method. So if you are looking for updates, check the copyColors() method

File: src/main/java/org/mineacademy/fo/model/SimpleScoreboard.java [hunk 1]
```diff
@@ -16,622 +12,556 @@
...
+
+    public SimpleScoreboard(final String title) {
+        this(title, 20);
+    }
+
+    /**
+     * Create a new scoreboard with customizable updateDelayTicks
+     */
+    public SimpleScoreboard(final String title, final int updateDelayTicks) {
+        //Scoreboards were introduced in 1.5, objectives were added in 1.8
```
Comment (Jakubk15) [inline src/main/java/org/mineacademy/fo/model/SimpleScoreboard.java:107]: Could you put a space after `//`? It just looks better, but that's just my opinion 😅

**Comment by kangarko** [comment]
Invalid formatting, I can't see what's changed, no pull

**Comment by Jakubk15** [comment]
Seems like he has to use the tab character, or anything that was there before.

**Comment by Tvhee-Dev** [comment]
Matej, seems that I had Tab disabled in this commit, but that my previous commit was with Tab character. That means that, after you have reviewed the changes, I will disable the Tab character in my IntelliJ and make another commit so you can pull

**Comment by kangarko** [comment]
Screenshot 2022-11-10 at 19 11 15

@Tvhee-Dev Please pull my Reformat commit and fix 1.8.8 and 1.7.10 (and possibly other older versions) colorizing

**Pull request merged**